pkg/component/component.go

```diff
@@ -66,28 +66,38 @@ func CreateFromGit(client *occlient.Client, name string, ctype string, url strin
 	// save source path as annotation
 	annotations := map[string]string{componentSourceURLAnnotation: url}
 
-	output, err := client.NewAppS2I(name, ctype, url, labels, annotations)
+	err = client.NewAppS2I(name, ctype, url, labels, annotations)
 	if err != nil {
-		return "", errors.Wrapf(err, "unable to create git component %s", name)
+		return errors.Wrapf(err, "unable to create git component %s", name)
 	}
-	return output, nil
+
+	//get the latest build name for following
+	buildName, err := client.GetLatestBuildName(name)
+	if err != nil {
+		errors.Wrap(err, "unable to create from git")
+	}
+	err = client.FollowBuildLog(buildName)
```

Comment:
please add the following message before showing logs, so  it is consistent with output when building from a local directory
```
	fmt.Println("please wait, building application...")
```

Reply:
@kadel shouldn't it be building component?